public/js/dashboardCategories.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * TODO: GET all categories from database
+ * @return JSON of all category entry information 
+ */
+function getAllCategories() {
+    var allCategories = [
+        {
+            "ID": "clothing",
+            "name": "Clothing",
+        }, {
+            "ID": "food",
+            "name": "Food",
+        }, {
+            "ID": "house-supplies",
+            "name": "Household Supplies",
+        }
+    ]
+    return allCategories;
+}
+
+/**
+ * TODO: GET an organizations accepted cagtegories from database
+ * @param {string} organizationID ID of organization requesting categories
+ */
+function getAllAcceptedCategories(organizationID) {
+    var allAcceptedCategories = [
+        {
+            "name": "Clothes",
+            "ID": "org-name-clothes",
+            "instructions": ["review quality checklist","unlock donation locker (code: 5248)","place in donation locker"],
+            "quality": ["no holes","gently used","washed recently"],
+        },
+        {
+            "name": "Food",
+            "ID": "org-name-food",
+            "instructions": ["review quality checklist","place in plastic/paper bag","bring inside during open hours"],
+            "quality": ["expiration date > 6 months from current date","no holes","labels on cans"],
+        },
+    ]
+    return allAcceptedCategories;
+}
+
+/**
+ * TODO: Add category to database
+ */
+function addCategory() {
+    console.log("added a category");
+}
+
+/**
+ * @param {string} categoryID ID of the category to be deleted.
+ */
+function deleteCategory(categoryID) {
+    console.log("deleted" + categoryID);
+}
+
+window.onload = async function() {
+    let categoriesTemplate = document.getElementById('categories-template').innerHTML;
+    const renderCategories = Handlebars.compile(categoriesTemplate);
+    const allCategories = await this.getAllCategories();
+    document.getElementById('categories').innerHTML = renderCategories({categoryList: allCategories});
+
+    let cardsTemplate = document.getElementById('cards-template').innerHTML;
+    const renderCards = Handlebars.compile(cardsTemplate);
+    const allCards = await this.getAllAcceptedCategories();
+    this.document.getElementById('card-holder').innerHTML = renderCards({cardList: allCards});
```

Comment:
nit: "card" wording by itself may be a little too general. Use naming that reflects its cards of the organizations accepted categories.